### PR TITLE
example(label): add an example of label creation from mouse click

### PR DESCRIPTION
**Pull request by mgermerie**
## Description

Add an example of label creation from mouse event.
When user clicks some position on the map while holding `p` key pressed, a popup created from a custom `div` will appear at the clicked position.

### CODE REVIEW

File: examples/label.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```
Comment (gchoqueux) [inline examples/label.html]: Could you rename the file to popup?

File: examples/label.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                const geometry = feature.bindNewGeometry();
+                // fill geometry with point
+                geometry.startSubGeometry(1, feature);
+                geometry.pushCoordinates(coordinates, feature);
+                geometry.updateExtent();
+
+                feature.updateExtent(geometry);
+                collection.pushFeature(feature);
+
+                return Promise.resolve(collection);
```
Comment (gchoqueux) [inline examples/label.html]: Promise isn't necessary

File: examples/label.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            }
+
+            function addFeatureFromMouseEvent(event) {
+                featureCoord.setFromVector3(view.getPickingPositionFromDepth(view.eventToViewCoords(event)));
+                console.log(featureCoord);
+                createFeatureAt(featureCoord).then(function _(features) {
+                    // the source of the feature layer
+                    let source = new itowns.FileSource({ features });
```
Comment (gchoqueux) [inline examples/label.html]: replace let by const

**Review by gchoqueux — COMMENTED** [review]
Could you change Label to have custom `content` parameter.
For user could be create custom div for the label.
So, change the example to use custom div.

**Comment by mgermerie** [comment]
This PR is rebased on PR #1570.

**Pull request closed without merging**